app/controllers/qsets_controller.rb

```diff
@@ -3,17 +3,28 @@ class QsetsController < ApplicationController
 
   # shows all qsets
   def index
-    @qsets = @qsets.order(:name)
-    @question_counts = Question.all.group(:qset_id).count
+      if current_user.org.nil?
+        redirect_to root_url, alert: "You are not part of an organization."
```

Comment:
This is a bit confusing if the user doesn't really know what she/he is doing, or follows a link to `/qsets` that someone else provided her/him. I think ultimately we'd want to send the user to an organization sign-up page, but if we don't have that yet then we might want to add 'Please contact user support.' and include a `mailto:` link to `support@askup.net`. I'll raise the lack of an organization select page as a user story.